doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -35,6 +35,46 @@ Ubuntu 16.04.4 LTS was used throughout this document, other versions such as
      sudo service ssh status
      sudo service ssh start
 
+Install build tools and dependencies
+*************************
```

Comment:
The heading underline must be at least as long as the title it’s under. Can you update it?

Reply:
> The heading underline must be at least as long as the title it’s under. Can you update it?
Got it, and I have adjusted the length.

Reply:
Thanks for your advice and I have updated the "tracked-on".